examples/java/core/imports/build.gradle

```diff
@@ -0,0 +1,44 @@
+group 'com.microsoft.bond'
+version '1.0'
+
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'com.microsoft.bond:gradle:+'
+    }
+}
+
+apply plugin: 'application'
+apply plugin: 'com.microsoft.bond.gradle'
+
+sourceCompatibility = 1.8
+mainClassName = 'com.microsoft.bond.examples.Imports'
+applicationDefaultJvmArgs = ['-ea']
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile files('../../../../java/core/build/libs/bond-5.0.0.jar')
+}
+
+compileBond {
+
+    bondfiles 'src/main/bond/schema.bond'
+
+    options '-n', 'Examples=com.microsoft.bond.examples.imports',
```

Comment:
Any thoughts on prioritizing adding these as first-class params to the plugin? It would be a single-sitting task to make this work:
```
compileBond {
  bondfiles 'whatever'
  mapNamespace 'Examples=com.microsoft.bond.examples.imports',
               'Common=com.microsoft.bond.examples.imports.common'
  importDir 'src/main/bond/common'
}
```